src/mongo/delegates/SearchDocuments.class.php

```diff
@@ -330,7 +330,7 @@ private function addValuesToTarget($values, $field, &$target)
         
 		if(count($values) > 0){						
 	        for ($i=0; $i<$limit; $i++) {	        	
-	            $v = utf8_encode($values[$i]); //todo: added by CJC
```

Comment:
I don't have a problem with this change, but it does raise the question as to why it was added in the first place.